fem/ceed/operator.hpp

```diff
@@ -33,9 +33,17 @@ class Operator : public mfem::Operator
 #endif
 
 public:
+#ifdef MFEM_USE_CEED
+   /// This class takes ownership of op and will delete it
+   Operator(CeedOperator op);
+#endif
    void Mult(const mfem::Vector &x, mfem::Vector &y) const override;
    void AddMult(const mfem::Vector &x, mfem::Vector &y) const;
    void GetDiagonal(mfem::Vector &diag) const;
+   mfem::Operator * SetupRAP(const mfem::Operator *Pi,
+                             const mfem::Operator *Po)
+   { return mfem::Operator::SetupRAP(Pi, Po); }
```

Comment:
Do you really need to redefine this method? Isn't it already inherited? Or wouldn't `using mfem::Operator::SetupRAP;` suffice?

Reply:
You're right, fixed.